src/main/java/org/jitsi_modified/impl/neomedia/codec/video/vp9/DePacketizer.java

```diff
@@ -84,9 +140,25 @@ public static boolean isStartOfFrame(byte[] buf, int off, int len)
             return isValid(buf, off, len) && (buf[off] & B_BIT) != 0;
         }
 
+        /**
+         * Sets the B bit (start of frame) in the first byte of the payload
+         * descriptor.
+         *
+         * @param buf the byte buffer that holds the VP9 payload.
+         * @param off the offset in the byte buffer where the VP9 payload starts.
+         * @param start the new value of the start of frame bit.
+         */
+        public static void setStartOfFrame(byte[] buf, int off, boolean start)
+        {
```

Comment:
I call `isValid` for all the `get` methods, and all the `set`s where the field is optional. I don't check it for the methods that just set a bit in the first byte.  Do you think I need to?

Reply:
Since `isValid` checks for things that would fail even when setting a bit in the first byte (buf being null, etc.) it seems like it should be checked even for these?

Reply:
Well, I guess the question is what should the `set` functions do on failure - if we'd just throw, it seems like the out-of-bounds write is just as good.  (Though I guess the buffer could be long enough, but not `len`?)

Reply:
Yeah I was commenting more for the consistency.  But, to your point, in the case where buffer is long enough but not `len` we definitely don't want to write.

Reply:
Did you end up changing anything here?

Reply:
I've changed it now.

(These functions were only used by the unit testing code, which is why I didn't worry about them before.)